App/TopTrumps/TopTrumps.Data/Helpers/ISqlHelper.cs

```diff
@@ -6,7 +6,7 @@ public interface ISqlHelper
     {
         string GetCardQuery(int id);
 
-        string GetAllCardsQuery();
+        string GetAllUsersQuery();
```

Comment:
Why are we renaming the `GetAllCardsQuery` to `GetAllUsersQuery` ?

Reply:
This was a mistake and now fixed